src/components/Primary-Button/Button.constants.ts

```diff
@@ -0,0 +1,58 @@
+import type { ButtonVariant, ButtonSize, ButtonShape } from './Button.types';
```

Comment:
Folder name uses a hyphen ('Primary-Button'). Project CDD conventions require PascalCase component folders (e.g. 'PrimaryButton/'). Please rename the folder to PascalCase to match CDD structure and update imports accordingly.

**Sugerencia:**
```typescript
Rename directory to src/components/PrimaryButton/ and update import paths in affected files.
```